easybuild/easyconfigs/t/TAU/TAU-2.22.2-gompi-1.4.12-no-OFED.eb

```diff
@@ -0,0 +1,34 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   New BSD
+#
+# This work is based from experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+name = 'TAU'
+version = '2.22.2'
+
+homepage = 'http://tau.uoregon.edu'
+description = """The TAU Performance System is a portable profiling and tracing toolkit
+ for performance analysis of parallel programs written in Fortran, C, C++, Java, Python."""
+
+toolchain = {'name': 'gompi', 'version': '1.4.12-no-OFED'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.cs.uoregon.edu/research/paracomp/tau/tauprofile/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('OTF', '1.12.4'),
+    ('PAPI', '5.2.0'),
+    ('PDT', '3.19'),
+    ('Scalasca', '1.4.3'),
+    ('Score-P', '1.2.1'),
```

Comment:
This is what you get, when you wait to merge a request for more than 2 years 😉

The TAU easyconfig will probably only work with an old version of Score-P which was based on Cube 3.X. Not sure what the last version of Score-P was that supported this.

Meanwhile, Scalasca-1.X (as well as VampirTrace) are very outdated; e.g. they no longer work with a MPI3 mpi library.
In my private TAU easyconfig, I have the following lines:

# compiler toolchain depencies
dependencies = [
    ('binutils', '2.22', "", ('GCC', '4.8.3')),
    ('OTF', '1.12.5'),
    ('PDT', '3.20'),
    #('Scalasca', '1.4.4'),      # obsolete backend, use Score-P
    ('Score-P', '1.4.2'),
    #('VampirTrace', '5.14.4'),  # obsolete backend, use Score-P
    ('PAPI', '5.4.1'),
]

You can either remove Scalasca-1.X (and VampirTrace) support, or change the comment to "obsolete backend for historical versions, use Score-P"

Bernd


From: Kenneth Hoste<mailto:notifications@github.com>
Sent: ‎Friday‎, ‎10‎ ‎June‎, ‎2016 ‎17‎:‎21
To: hpcugent/easybuild-easyconfigs<mailto:easybuild-easyconfigs@noreply.github.com>
Cc: Bernd Mohr<mailto:b.mohr@fz-juelich.de>, Mention<mailto:mention@noreply.github.com>


In easybuild/easyconfigs/t/TAU/TAU-2.22.2-gompi-1.4.12-no-OFED.eb<https://github.com/hpcugent/easybuild-easyconfigs/pull/509#discussion_r66631880>:

> +homepage = 'http://tau.uoregon.edu'
> +description = """The TAU Performance System is a portable profiling and tracing toolkit
> + for performance analysis of parallel programs written in Fortran, C, C++, Java, Python."""
> +
> +toolchain = {'name': 'gompi', 'version': '1.4.12-no-OFED'}
> +toolchainopts = {'usempi': True}
> +
> +source_urls = ['http://www.cs.uoregon.edu/research/paracomp/tau/tauprofile/dist/']
> +sources = [SOURCELOWER_TAR_GZ]
> +
> +dependencies = [
> +    ('OTF', '1.12.4'),
> +    ('PAPI', '5.2.0'),
> +    ('PDT', '3.19'),
> +    ('Scalasca', '1.4.3'),
> +    ('Score-P', '1.2.1'),


@berndmohr<https://github.com/berndmohr> maybe you have an idea on how to resolve the above?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/hpcugent/easybuild-easyconfigs/pull/509/files/b68ebcfbba2e3a2710aa7a09cd999945543a2efb#r66631880>, or mute the thread<https://github.com/notifications/unsubscribe/AFfSryu4CKMzIvV3Iqgx2rz7t-SpY-gCks5qKYEMgaJpZM4BMAWU>.


------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
Forschungszentrum Juelich GmbH
52425 Juelich
Sitz der Gesellschaft: Juelich
Eingetragen im Handelsregister des Amtsgerichts Dueren Nr. HR B 3498
Vorsitzender des Aufsichtsrats: MinDir Dr. Karl Eugen Huthmacher
Geschaeftsfuehrung: Prof. Dr.-Ing. Wolfgang Marquardt (Vorsitzender),
Karsten Beneke (stellv. Vorsitzender), Prof. Dr.-Ing. Harald Bolt,
Prof. Dr. Sebastian M. Schmidt
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------